lib/Config/Populator.php

```diff
@@ -549,6 +551,13 @@ private function scheduleAuthorizedSendingEmailsCheck() {
     );
   }
 
+  private function initLastAnnouncementDate() {
```

Comment:
I think we should rather schedule the task instead of getting the date directly here. It can take some time to load the Beamer API and our plugin activation/upgrade is already slow. We should keep it as fast as possible.